Formula/l/lnav.rb

```diff
@@ -41,11 +43,15 @@ class Lnav < Formula
   def install
     system "./autogen.sh" if build.head?
     ENV.append "LDFLAGS", "-L#{Formula["libarchive"].opt_lib}"
+    ENV.append "LDFLAGS", "-L#{Formula["ncurses"].opt_lib}"
+    ENV.append "CXXFLAGS", "-I#{Formula["ncurses"].opt_include}"
```

Comment:
Can autogen not detect these?

Reply:
I'm not sure, maybe a question for @tstack if they recall why this had to be plumbed in for `libarchive` (5194fe31057ab6e100c6db3006856b308321fd2b). If you mean literally `autogen.sh` — it's only run if `build.head?`, so it wouldn't apply for regular installs, I think.

I can try installing without these and see if it picks up the right version automatically.

Reply:
There's a `--with-ncurses` flag you can pass to configure.  But, it's not much different from adding the path to the `LDFLAGS` and `CXXFLAGS`.

Reply:
It is much easier to read for maintainers though, so let's use that

Reply:
Sounds good. In that case is `libarchive` still needed in `LDFLAGS` too, or should I also remove that while I'm in there?

Reply:
If it works without the explicit `LDFLAGS` (which I presume it should, given the `--with-libarchive` flag below): yes, let's remove that for the same reason.